### ISSUE TITLE
Too little documentation

### ISSUE THREAD

**Issue by sergei-mironov** [felixr/urxvt-theme-switch#6]
Hi! The plugin looks promising, but I can't find any description regarding the themes themselves. Could you please drop a note on where to find them? Regards!

**Comment by ep12** [comment]
That's absolutely right. It is unlucky that one has to understand the source code. I don't know perl but I managed to find out that it is basically the .Xressources syntax:
```
foreground: #373b41
background: #ffffff
cursorColor: #373b41

color0:  #1d1f21
color8:  #969896

color1:  #cc342b
color9:  #cc342b

color2:  #198844
color10: #198844

color3:  #fba922
color11: #fba922

color4:  #3971ed
color12: #3971ed

color5:  #a36ac7
color13: #a36ac7

color6:  #3971ed
color14: #3971ed

color7:  #c5c8c6
color15: #ffffff
```
(Google light from [terminal.sexy](https://terminal.sexy))

The colors 0-15 are the typical ones: black, red, green, yellow, blue, magenta, cyan, grey/white
in a dark shade (0-7) and a light shade (8-15).

Apart from `foreground`, `backgroun`, `cursorColor` and color* the following settings are supported:
+ `colorBD` (bold)
+ `colorUL` (underline)
+ `colorBL` (blink)
+ `colorRV` (reverse)
+ `pointerColor`
+ `highlightColor` (selection)
+ `backgroundPixmap`
+ `italicColor`
+ `tintColor`
+ `boldColor`
+ `underlinedColor`
+ `borderColor`
+ `font`
+ `boldFont`
+ `italicFont`
+ `boldItalicFont`
Those are probably case-sensitive. For more information, consult `man urxvt`.
